install/ingest_sample_data.py

```diff
@@ -0,0 +1,45 @@
+import os
+from sqlalchemy import create_engine, text
+from sqlalchemy.orm import sessionmaker
+
+from opencdms.adapters import opencdmsdb
+
+
+# set connection details
+UID = os.environ["POSTGRES_USER"]
+PWD = os.environ["POSTGRES_PASSWORD"]
+DBNAME = os.environ["POSTGRES_DB"]
+DBHOST = os.environ["POSTGRES_HOST"]
+DBPORT = os.environ["POSTGRES_PORT"]
+DATADIR = os.environ["OPENCDMS_SAMPLE_DATA"]
+
+engine = create_engine(f"postgresql+psycopg2://{UID}:{PWD}@{DBHOST}:{DBPORT}/{DBNAME}", echo=True)
+
+opencdmsdb.start_mappers()
+
+session = sessionmaker(bind=engine)()
+cursor = session.connection().connection.cursor()
+
+# note, order is important
+data_tables = {
+    "cdm.host": ["hosts.csv"],
+    "cdm.source": ["source.csv"],
+    "cdm.observer": ["observer.csv"],
+    "cdm.deployment": ["deployment.csv"],
+    "cdm.observer_characteristics": ["observer_characteristics.csv"],
+    "cdm.observation": ["CA_6016527_1990.csv"],
```

Comment:
When I deploy online I'm ingesting all years. Should we make it easier to chose between one year or all years (and which should be the default)?

I think `CA_*.csv` works with the copy command if that's helpful.

Reply:
We're copying from STDIN rather than a file (with STDIN coming from python) so the CA_*.csv won't work. I'll add the extra files to the array.